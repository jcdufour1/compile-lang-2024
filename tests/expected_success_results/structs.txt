9/2
8/7
11/10
8/10
14/34
14/34
11/10
8/10
token: 923 hello
14/34
831/700
11/10
82 bye 56/84
71 bye 56/84
82 bye 56/84
82
