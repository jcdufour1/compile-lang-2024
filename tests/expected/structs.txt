9/2
8/7
11/10
8/10
14/34
14/34
87/23
87/23
